content/en/news/releases/1.6.x/announcing-1.6/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Change Notes	title: Change Notes
+description: Istio 1.6 release notes.
+weight: 10
+---
+
+## Traffic Management
+- ***Added*** experimental support for the Kubernetes Service APIs.
+- ***Improved*** support for Kubernetes Ingress, adding support for reading certificates from Secrets, pathType, and IngressClass.
+- ***Fixed*** a bug blocking external HTTPS/TCP traffic in some cases.
```

Comment:
should have a link from the doc

Reply:
I'll do this for all links in the doc. Good suggestion.